frontend/src/types/vendor.ts

```diff
@@ -57,7 +58,24 @@ export type Vendor = Pick<BackendVendor, 'id'
   'tags': VendorTag[]
 };
 
+export enum VendorSpecialty {
+  SPECIALTY_HAIR,
+  SPECIALTY_MAKEUP
+}
+
+function mapTagToSpecialty(tag: VendorTag): VendorSpecialty | null {
```

Comment:
there's also the tag to id map in TagSelector. Can the maps be in the same place for easier maintenance?